package.json

```diff
@@ -23,6 +23,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.8.1",
     "@fortawesome/react-fontawesome": "^0.1.4",
     "@hapi/joi": "^15.1.1",
+    "@material-ui/core": "^4.4.3",
```

Comment:
is it needed to install a whole library to use one component? we will discuss using material UI tomorrow and decide handling components in general

Reply:
Thanks for taking time to review. I believe we will use material ui in all components, and here I started 😀😁